leftwm/src/bin/leftwm-command.rs

```diff
@@ -12,15 +12,10 @@ async fn main() -> Result<()> {
         .version(env!("CARGO_PKG_VERSION"))
         .about("Sends external commands to LeftWM")
         .arg(
-            arg!("command").help("The command to be sent. See 'list' flag."), // .required(true)
-                                                                              // .multiple(true)
-        )
-        .arg(
-            arg!("list")
-                .help("Print a list of available commands with their arguments.")
-                .short('l')
-                .long("list"),
+            arg!(--"command" [COMMAND] "The command to be sent. See 'list' flag."), // .required(true)
```

Comment:
This is wrong, we don't want to `leftwm-command --command "Command"` but `leftwm-command "Command"`. 
```suggestion
            arg!(name:command [COMMAND] "The command to be sent. See 'list' flag."),
```